src/clr-angular/data/datagrid/datagrid-column-toggle-button.ts

```diff
@@ -10,35 +10,19 @@ import { ColumnToggleButtons, ColumnToggleButtonsService } from './providers/col
 @Component({
   selector: 'clr-dg-column-toggle-button',
   template: `
-        <button
+        <button class="btn btn-sm btn-link p6 text-uppercase"
             (click)="click()"
-            [disabled]="toggleButtons.selectAllDisabled && !isOk()"
-            [ngClass]="getClasses()"
+            [disabled]="toggleButtons.selectAllDisabled"
             type="button">
             <ng-content></ng-content>
         </button>
     `,
-  host: { '[class.action-right]': 'isOk()' },
 })
 export class ClrDatagridColumnToggleButton {
-  @Input() clrType: ColumnToggleButtons;
+  @Input() clrType: ColumnToggleButtons = 'selectAll';
```

Comment:
It seems weird to have this defined as a type `ColumnToggleButtons` and then hardcoded as the `selectAll` string. Can `clrType` ever be something that _isn't_ `"selectAll"`?

If it can, then it seems we would have a typing issue. If not, it seems weird to have it be an input.

Reply:
Discussed this briefly with the dev team. We thought it'd be good to preserve backward compatibility even though we're removing the other type so that this is still a valid mark up:

`<clr-dg-column-toggle-button clrType="selectAll">Alle auswählen!</clr-dg-column-toggle-button>`

As well as one without which would default to the selectAll type:
`<clr-dg-column-toggle-button>Alle auswählen!</clr-dg-column-toggle-button>`

Reply:
What Jeeyun said. Please just make sure to:
- Add an `@deprecated` JSDoc annotation,
- Remove the `clrType` completely from the docs.